#!/usr/bin/env python
from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("max_distance", double_t, 0, "Max distance filter (m)", 30.0, 0.0, 100.0)
gen.add("eps", double_t, 0, "DBSCAN radius (m)", 0.10, 0.0, 2.0)
gen.add("min_samples", int_t, 0, "DBSCAN min samples", 10, 1, 100)
gen.add("downsample_n", int_t, 0, "Downsample every N points", 1, 1, 100)

exit(gen.generate("auv_mapping", "slalom_clustering", "SlalomClustering"))
